dataland_analytics/analysis_scripts/scripts_with_api_read/check_for_dataset.py

```diff
@@ -0,0 +1,63 @@
+# Check for a list of companies if SFDR data sets exists for reporting periods 2022 and 2023
+
+import dataland_datasets
+import pandas as pd
+
+from dataland_analytics.config import *
+
+configuration = dataland_datasets.Configuration(
+    host=DATALAND_SERVER + "api", access_token=PERSONAL_USER_API_KEY
+)
+
+company_ids = [
+    "a3b02fec-d9db-4af8-8698-3b2c7ec6e396",
+    "61615c24-747b-431d-91b5-798a38f42a17",
+    "58a603dc-f53c-4499-add1-1c463b129aba",
+]
```

Comment:
The ids should be removed.

Reply:
I intentionally kept in a few IDs to make it clearer how the Script should be used. The list was actually about 200 IDs which I cut down to 3 as a minimum working example.